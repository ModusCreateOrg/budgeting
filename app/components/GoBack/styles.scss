.containerGoBack {
  cursor: pointer;
  display: inline;
  border: 1px solid black;
  width: 70px;
  margin: 5px;
}