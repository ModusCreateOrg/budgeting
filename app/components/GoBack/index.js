// @flow
import React from 'react';
import styles from './styles.scss';

type GoBackProps = {
  goBack: Function,
  text: string,
};

const GoBack = (props : GoBackProps) => (
  <div className={styles.containerGoBack} onClick={props.goBack} role="button" tabIndex="0">
    <span>{props.text}</span>
  </div>
);

export default GoBack;
