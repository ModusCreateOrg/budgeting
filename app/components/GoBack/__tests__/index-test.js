import React from 'react';
import renderer from 'react-test-renderer';
import GoBack from '../';

it('renders correctly', () => {
  const tree = renderer.create(<GoBack text="← Back" goBack={() => {}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
