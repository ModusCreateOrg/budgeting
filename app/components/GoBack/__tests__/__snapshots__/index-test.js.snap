// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="containerGoBack"
  onClick={[Function]}
  role="button"
  tabIndex="0"
>
  <span>
    ‚Üê Back
  </span>
</div>
`;
